src/renderer/pages/Staking/Overview/components/NominatorsList/NominatorsList.tsx

```diff
@@ -68,36 +33,52 @@ export const NominatorsList = ({
   onToggleNominator,
 }: Props) => {
   const { t } = useI18n();
+  const { getNextUnstakingEra, hasRedeem } = useStakingData();
+
+  const getUnstakeBadge = (stake: NominatorInfo) =>
```

Comment:
I would suggest to use curly brackets for readability
Same for `getRedeemBadge`

```javascript
const getUnstakeBadge = (stake: NominatorInfo, era?: number): ReactNode | undefined => {
  const nextUnstakingEra = getNextUnstakingEra(stake.unlocking, era);
  if (!nextUnstakingEra) return undefined;

  return (
    <Tooltip offsetPx={-65} content={<Trans t={t} i18nKey="staking.tooltips.unstakeDescription" />}>
        <div className="flex gap-x-1 items-center rounded-md bg-badge-background px-2 py-0.5">
          <Icon name="unstake" className="text-icon-accent" size={14} />
          <HelpText className="text-icon-accent">
            <TimeToEra className="my-1" api={api} era={nextUnstakingEra} />
          </HelpText>
        </div>
      </Tooltip>
  );
}
```